deploy/helm/kuberhealthy/values.yaml

```diff
@@ -128,3 +128,4 @@ checkReaper:
     repository: kuberhealthy/check-reaper
     tag: v1.4.0
   nodeSelector: {}
+  startingDeadlineSeconds: # Default is nil for CronJobs
```

Comment:
similar to how we have it for deployments (https://github.com/Comcast/kuberhealthy/blob/058a6b3a3eb488547199809030aede878b13b632/deploy/helm/kuberhealthy/values.yaml#L38-L39) could you comment out this `startingDeadlineSeconds:`?

Reply:
Thanks for pointing this out. Forgot to update these values